### ISSUE TITLE
Support monorepos like TurboRepo.

### ISSUE THREAD

**Issue by chrisbull** [antfu/eslint-config#640]
https://github.com/antfu/eslint-config/blob/20dd434ad89d6994fcd114b362bf0b8b16de68df/src/configs/react.ts#L106

Issue is that NextJS isn't detected when using a monorepo.

I think local-pkg doesn't recognize that I have "next" in my ./apps/web/package.json so eslint is throwing errors for `metadata`

**Comment by antfu** [comment]
Couldn't you also install `next` in your top-level package.json?

**Comment by pauliesnug** [comment]
> Couldn't you also install `next` in your top-level package.json?

Yes, this is the recommended solution. Build tools and core libraries should be installed in the top-level Turborepo `package.json` file anyways.

**Comment by huangliangb0** [comment]
Yes, it can be installed on the top floor of the project. However, it seems impossible for me to configure and deploy it to the npm private library to make it the public configuration of eslint for multiple of my projects

**Comment by huangliangb0** [comment]
> 你不能`next`在顶层 package.json 中安装吗？

Yes, it can be installed on the top floor of the project. However, it seems impossible for me to configure and deploy it to the npm private library to make it the public configuration of eslint for multiple of my projects。
